### ISSUE TITLE
[Bug]: deb/rpm packages use custom `netdata.conf` that provides no value

### ISSUE THREAD

**Issue by ilyam8** [netdata/netdata#12530]
### Bug description

Using [custom netdata.conf](https://github.com/netdata/netdata/blob/master/system/netdata.conf) no longer relevant after #12339.

Problems:
- different experience depends on the installation method
  - static, from source: "netdata.conf" with the actual config with all available options.
  - native packages: "netdata.conf" almost empty config.
- user have to download "netdata.conf" from http://localhost:19999/netdata.conf in order to change/tune configuration. 

### Expected behavior

- deb/rpm packages don't use [custom netdata.conf](https://github.com/netdata/netdata/blob/master/system/netdata.conf).
- we should download the actual configuration file right after the installation process (as we do when installing static builds, from the source).

### Steps to reproduce

1. install deb/rpm package
2. check /etc/netdata/netdata.conf

### Installation method

kickstart.sh

### System info

```shell
$ uname -a; grep -HvE "^#|URL" /etc/*release
Linux deb-fra1-01 5.10.0-13-amd64 #1 SMP Debian 5.10.106-1 (2022-03-17) x86_64 GNU/Linux
/etc/cloud-release:ID="generic"
/etc/cloud-release:VERSION="20220202-906"
/etc/os-release:PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
/etc/os-release:NAME="Debian GNU/Linux"
/etc/os-release:VERSION_ID="11"
/etc/os-release:VERSION="11 (bullseye)"
/etc/os-release:VERSION_CODENAME=bullseye
/etc/os-release:ID=debian
```

### Netdata build info

```shell
$ /usr/sbin/netdata -W buildinfo
Version: netdata v1.33.1-236-nightly
Configure options:  '--build=x86_64-linux-gnu' '--includedir=${prefix}/include' '--mandir=${prefix}/share/man' '--infodir=${prefix}/share/info' '--disable-option-checking' '--disable-silent-rules' '--libdir=${prefix}/lib/x86_64-linux-gnu' '--libexecdir=${prefix}/lib/x86_64-linux-gnu' '--disable-maintainer-mode' '--disable-dependency-tracking' '--prefix=/usr' '--sysconfdir=/etc' '--localstatedir=/var' '--libdir=/usr/lib' '--libexecdir=/usr/libexec' '--with-user=netdata' '--with-math' '--with-zlib' '--with-webdir=/var/lib/netdata/www' 'build_alias=x86_64-linux-gnu' 'CFLAGS=-g -O2 -ffile-prefix-map=/usr/src/netdata=. -fstack-protector-strong -Wformat -Werror=format-security' 'LDFLAGS=-Wl,-z,relro' 'CPPFLAGS=-Wdate-time -D_FORTIFY_SOURCE=2' 'CXXFLAGS=-g -O2 -ffile-prefix-map=/usr/src/netdata=. -fstack-protector-strong -Wformat -Werror=format-security'
Install type: binpkg-deb
    Binary architecture: x86_64
    Packaging distro:
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES
    ACLK Next Generation:       YES
    ACLK-NG New Cloud Protocol: YES
    ACLK Legacy:                NO
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         YES
Libraries:
    protobuf:                YES (system)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    YES
    EBPF:                    YES
    IPMI:                    YES
    NFACCT:                  YES
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 NO
    Prometheus Remote Write: YES
```

**Comment by Ferroin** [comment]
Closing as a duplicate of https://github.com/netdata/netdata/issues/12404.
